clusterloader2/pkg/test/modifier_test.go

```diff
@@ -0,0 +1,45 @@
+package test
+
+import (
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+var _ = ginkgo.Describe("Modifier", func() {
```

Comment:
Let's not introduce ginkgo to our tests - it's simple unit test and ;et's stick to basic unit tests please

Reply:
Removed ginkgo